CareKitStore/CareKitStore/CoreData/OCKStore.swift

```diff
@@ -292,6 +295,18 @@ open class OCKStore: OCKStoreProtocol, Equatable {
         return true
     }
 
+    private func updateFileProtectionPathAtURL(_ url: URL) throws {
+
+        #if os(macOS)
```

Comment:
Currently only needed for macOS, but this method calls through just incase it's needed for other OS's later